test/TesterInternal/GeoClusterTests/LogConsistencyTestFixture.cs

```diff
@@ -203,6 +207,16 @@ public virtual void Dispose()
 
         private const int Xyz = 333;
 
+        private void AssertEqual<T>(T expected, T actual, string grainidentity)
```

Comment:
Generally we use lower camel case for instance fields, variables, and parameters, so `grainidentity` would be `grainIdentity`.

Reply:
yep.